src/blockchain-utilities/derivation.ts

```diff
@@ -75,17 +79,15 @@ export const fromBytes32String = (input: string) => ethers.utils.parseBytes32Str
 /**
  * Derives a sponsor wallet, given a mnemonic and dapiName.
  *
- * @param sponsorWalletMnemonic the sponsor wallet mnemonic
+ * @param airnodeMnemonic the sponsor wallet mnemonic
```

Comment:
Update "the sponsor wallet mnemonic"? Perhaps for completeness, add the new `protocolName` `@param`?

Reply:
Thanks. Although I am not sure if this documentation is not an overkill. Seems like all of what's written can be derived from the parameter names/types.